packages/config/src/chainConfig/networks/sepolia.ts

```diff
@@ -4,7 +4,7 @@ import {ChainConfig} from "../types.js";
 import {chainConfig as mainnet} from "../configs/mainnet.js";
 
 // Sepolia beacon chain config:
-// https://github.com/eth-clients/sepolia/blob/main/bepolia/config.yaml
+// https://github.com/eth-clients/sepolia/blob/main/metadata/config.yaml
```

Comment:
We are still pulling sepolia network config from the merge-testnets repo
https://github.com/ChainSafe/lodestar/blob/8b6ecc44c1892115d0e7396b2a5f0ae8fc0d9e6f/packages/cli/src/networks/sepolia.ts#L5-L6

is this something we wanna update in this PR as well?

Reply:
yes please. The merge-testnets repo has been archived now.